tailwind.config.js

```diff
@@ -1,12 +1,6 @@
 module.exports = {
-  purge: {
-    content: ["./src/**/*.{js,jsx,ts,tsx}"],
-  },
-  darkMode: false, // or 'media' or 'class'
+  content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    screens: {
-      short: { raw: "(max-height: 615px)" },
-    },
```

Comment:
This is being used in the upload screen. What's the reason for taking this out?

Reply:
That search-fu has failed me to find reference to it. Will revert that part.